setup/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "setup-notation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Duffney/setup-notation.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Duffney/setup-notation/issues"
+  },
+  "homepage": "https://github.com/Duffney/setup-notation#readme",
```

Comment:
Same here.

Reply:
Remove this line because I didn't see it in the other actions like sign and verify.